CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ project(rsgain
   LANGUAGES CXX
 )
 
+option(SCALEPROGBAR "Scale progress bar to column width" ON)
```

Comment:
No need to have the on/off option. We can check for the existence of the width variable to determine the state of the setting